unity/Assets/Scripts/Content/DictionaryI18n.cs

```diff
@@ -262,7 +262,7 @@ public void flushRaw()
             {
                 string key = rawLine.Split(COMMA)[0];
                 // Process non repeated list line
-                if (!dict.ContainsKey(key))
+                if (key != "." && !dict.ContainsKey(key))
```

Comment:
Sorry but I'm creating the editabletextBox with special icons and trying to multiline the field and this issue was "on my way". 
I'm not sure that with this condition all duplicated languages list entry wil dissapear. So I didn't consider this issue fixed.